universal/containers/AppContainer.js

```diff
@@ -11,6 +11,7 @@ class AppContainer extends React.Component {
 
   componentDidMount () {
     const { history, store } = this.props
+    console.log(store)
```

Comment:
Did you mean to leave this? I can't tell if it's related to the new `redux-logger`.